### ISSUE TITLE
Support for monospace index entries

### ISSUE THREAD

**Issue by mattharrison** [RolfBremer/in-dexter#20]
In my coding books, I want to have an index entry typeset in monospace font. This represents code, arguments, or variables.

I use 

```
#index()[`conda`]
```

but there is not monospace font entry in the index.

**Comment by RolfBremer** [comment]
You can use the display parameter of the index function:

```typ
#index-main(display: `conda`, "conda")
```

Hope that helps.

**Comment by mattharrison** [comment]
Thanks for the hint. I ended up wrapping with []

```typ
#index(display: [`conda`])[`conda`]
```

but the content still comes out with the normal typeface.

**Comment by RolfBremer** [comment]
We have fixed that in version 0.5.3 in the Github main branch. It is also submitted to the Typst Universe as a Package, and should be available soon. Thanks for reporting!

**Comment by mattharrison** [comment]
Thanks so much for the update. I have had a chance to dive in yet, but I appreciate your work!
